providers.tf

```
@@ -4,66 +4,87 @@
 
 provider "aws" {
   region = var.aws_region
+  default_tags {
+    tags = var.tags
+  }
```

Consistent with the changes in the other providers.

```suggestion
  default_tags {
    tags = var.tags
  }
  region = var.aws_region
```